lib/msf/core/handler/reverse.rb

```diff
@@ -82,12 +83,28 @@ def setup_handler
                 'MsfPayload' => self,
                 'MsfExploit' => assoc_exploit
               })
+
+          #Checking whether LHOST is a loopback address
+
+          def is_loopback_address(address)
+            begin
+               a = IPAddr.new(address.to_s)
+               return true if IPAddr.new('127.0.0.1/8') === a
+               return true if IPAddr.new('::1') === a
+            rescue
+            end
+            return false
+          end
+
+           if is_loopback_address(ip)
```

Comment:
Sorry, jumped the gun a bit. This is fixed in my most recent commit. I'm pushing up a change that removes the newlines as well.